### PR TITLE
feat: Add the accessibility page

### PR DESCRIPTION
**Pull request by spydon**
image

No functionality yet, just the initial design implementation.

### CODE REVIEW

File: packages/ubuntu_provision/lib/src/network/network_page.dart [hunk 1]
```diff
@@ -33,7 +33,7 @@ class NetworkPage extends ConsumerWidget with ProvisioningPage {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final model = ref.watch(networkModelProvider);
-    final lang = NetworkLocalizations.of(context);
+    final lang = UbuntuProvisionLocalizations.of(context);
```
Comment (d-loose) [inline packages/ubuntu_provision/lib/src/network/network_page.dart:36]: Probably unintended - we should cleanup the l10n stuff separately :)

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_en.arb [hunk 2]
```diff
@@ -1,4 +1,28 @@
 {
+  "accessibilityPageTitle": "Accessibility in {DISTRO}",
+  "@accessibilityPageTitle": {},
```
Comment (d-loose) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_en.arb:3]: could we add
```
    "type": "text",
    "placeholders": {
      "DISTRO": {}
    }
```
here?

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_en.arb [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+  "accessibilityPageBody": "Customise Ubuntu to your needs before you set up. You can change them later in System Settings.",
```
Comment (d-loose) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_en.arb:4]: "Ubuntu" should be "{DISTRO}"

File: packages/ubuntu_provision/lib/src/accessibility/accessibility_page.dart [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final model = ref.watch(accessibilityModelProvider);
+    final lang = UbuntuProvisionLocalizations.of(context);
+    final scrollBarPadding =
+        (ScrollbarTheme.of(context).thickness?.resolve({}) ?? 6) * 4;
+
+    return HorizontalPage(
+      windowTitle: lang.accessibilityPageTitle(model.distroName),
+      title: lang.accessibilityPageTitle(model.distroName),
```
Comment (d-loose) [inline packages/ubuntu_provision/lib/src/accessibility/accessibility_page.dart:26]: Could we use the `flavorProvider` instead? Then we wouldn't need the `distroName` property and the `ProductService` in the view model

Comment (spydon) [reply packages/ubuntu_provision/lib/src/accessibility/accessibility_page.dart]: Aah, much better!

**Review by d-loose — APPROVED** [review]
LGTM :+1:

**Pull request merged**